packages/e2e-tests/specs/reusable-blocks.test.js

```diff
@@ -42,7 +42,7 @@ describe( 'Reusable Blocks', () => {
 
 		// Wait for creation to finish
 		await page.waitForXPath(
```

Comment:
Should it be wrapped with an e2e test util to simplify usage in different places? It might be too early, but at least I can see the value of having a local function which simplifies his logic.

Reply:
Ya I agree, it'd be handy to have a test util for verifying notices appear (would be good if it allows for notice type as well as content in the signature).